sonar-project.properties.license

```diff
@@ -0,0 +1,7 @@
+                  
```

Comment:
You can remove the `sonar-project.properties` file. And therefore also remove this file.